odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -50,27 +196,44 @@ def fbp_op(ray_trafo, padding=True):
         `CircularConeFlatGeometry` : Approximate reconstruction, correct in
         limit of fan angle = 0 and cone angle = 0.
 
-        `HelicalConeFlatGeometry` : Very approximate.
+        `HelicalConeFlatGeometry` : Very approximate unless a
+        `tam_danielson_window` is used. Accurate with the window.
```

Comment:
I used [this code](https://gist.github.com/adler-j/d1eae02978296ed710994e8f5359676b) to generate an example image showing the artefact:

![windowed_filtered_back-projection](https://cloud.githubusercontent.com/assets/2202312/21157523/e70ed6f6-c179-11e6-8be5-24bcca8ed58c.png)